great_expectations/datasource/fluent/pandas_file_path_datasource.pyi

```diff
@@ -0,0 +1,26 @@
+from logging import Logger
+from typing import ClassVar, Dict, List, Type
+
+from great_expectations.datasource.fluent.file_path_data_asset import _FilePathDataAsset
+from great_expectations.datasource.fluent.interfaces import DataAsset as DataAsset
+from great_expectations.datasource.fluent.pandas_datasource import _PandasDatasource
+
+logger: Logger
+
+class CSVAsset(_FilePathDataAsset): ...
+class ExcelAsset(_FilePathDataAsset): ...
+class JSONAsset(_FilePathDataAsset): ...
+class ORCAsset(_FilePathDataAsset): ...
+class ParquetAsset(_FilePathDataAsset): ...
+class FeatherAsset(_FilePathDataAsset): ...
+class HDFAsset(_FilePathDataAsset): ...
+class HTMLAsset(_FilePathDataAsset): ...
+class PickleAsset(_FilePathDataAsset): ...
+class SASAsset(_FilePathDataAsset): ...
+class SPSSAsset(_FilePathDataAsset): ...
+class STATAAsset(_FilePathDataAsset): ...
+class XMLAsset(_FilePathDataAsset): ...
```

Comment:
What purpose does this serve?

Reply:
This lets' `mypy` and the IDE properly understand these types. The pydantic generate model doesn't properly pass the type information.

We could do this better and also define all the specific attrs each asset has, but at least this lets the IDE and mypy know about the shared attributes that come from `_FilePathDataAsset`